examples/twilio_api_v2010.yaml

```diff
@@ -102,6 +102,205 @@ info:
   version: 1.11.0
 openapi: 3.0.1
 paths:
+  /v1/FlexFlows:
```

Comment:
Is this new path needed? Or rather, what in the new path are we testing and can it be merged to an existing path/operation? I prefer we keep this sample doc as tight as is needed given that more paths create more generated code which requires more (possibly duplicate) tests to keep coverage up.

Reply:
This is a non-v2010 path, adding it to an existing v2010 path might be confusing. I'd say let's keep this and add (in future, if need be) other non-v2010 api's under this path ?
A part of this testing was also to confirm if the change works fine e2e.